### ISSUE TITLE
Problem with inverse electron densities - no packets are escaping

### ISSUE THREAD

**Issue by unoebauer** [tardis-sn/tardis#419]
Some weird behaviour in the new plasma calculation or/and in the new layout or the ```montecarlo.pyx``` file lead to the following severe problem:

During each montecarlo packet propagation cycle, not a single packet escaped through the outer boundary of the computational domain. It seems that the simulation thinks that the distance to the next electron scattering event is zero. Thus, all injected packets immediately scatter and are eventually lost at the inner boundary. Somehow, probably somewhere in the montecarlo.pyx file the inverse electron densities are messed up (potentially pointing to uninitialised memory).

**Comment by unoebauer** [comment]
After updating to the current master AND updating the pandas library to pandas 0.16.2, the problem seems to have disappeared. 

However, @chvogl seems to have run into the same problem with the current master.

**Comment by wkerzendorf** [comment]
@unoebauer so you do not get a problem anymore with the current master and a new pandas version. If you have the machinery running to check for missing references specifically in `init_storage_model`, then we can make sure these are fixed. let me know, which ones these are. I have had a similar problem before, but I fixed it before merging. It resulted in the full tardis test failing.

**Comment by chvogl** [comment]
I encountered the same problem, when I tried to add some additional data for the continuum implementation in the __initalize_storage_model__  function.
For unknown reasons this resulted in the cleanup of the __inverse_electron_densities__ by the garbage collector. As a result the pointer to the inverse electron densities pointed to values of the order 1e-300.
As a consequence the distance to the next continuum event was always zero.
